src/api/application/application.ts

```diff
@@ -74,28 +83,37 @@ export class Application extends Base {
     }
 
     /**
-      Determines if the application is currently running.
-      @return {Promise.<boolean>}
+      * Determines if the application is currently running.
+      * @return {Promise.<boolean>}
     */
     public isRunning(): Promise<boolean> {
         return this.wire.sendAction('is-application-running', this.identity)
             .then(({ payload }) => payload.data);
     }
 
     /**
-      Closes the application and any child windows created by the application.
-      @param { boolean } force sets the value force to false
-      @return {Promise.<boolean>}
+      * Closes the application and any child windows created by the application.
+      * @param { boolean } force sets the value force to false
+      * @return {Promise.<boolean>}
     */
     public close(force: boolean = false): Promise<void> {
         return this.wire.sendAction('close-application', Object.assign({}, this.identity, {force})).then(() => undefined);
     }
 
+    /**
+     * Retrieves an array of wrapped fin.desktop.Windows for each of the application’s child windows.
+     * @return {Promise.<_Window>}
```

Comment:
https://stackoverflow.com/questions/14611995/how-to-specify-an-array-of-objects-as-a-parameter-or-return-value-in-jsdoc#14854851